tests/test_marker_definitions.py

```diff
@@ -1,34 +1,37 @@
-import pytest
-
+from sanity_html.dataclasses import Block, Span
 from sanity_html.marker_definitions import (
-    render_comment_marker,
-    render_emphasis_marker,
-    render_link_marker,
-    render_strong_marker,
+    CommentMarkerDefinition,
+    EmphasisMarkerDefinition,
+    LinkMarkerDefinition,
+    StrongMarkerDefinition,
 )
 
 sample_texts = ['test', None, 1, 2.2, '!"#$%&/()']
 
 
 def test_render_emphasis_marker_success():
     for text in sample_texts:
-        assert render_emphasis_marker(text) == f'<em>{text}</em>'
+        node = Span(**{'_type': 'span', 'text': text})
+        block = Block(_type='block', children=[node])
+        assert EmphasisMarkerDefinition.render(node, 'em', block) == f'<em>{text}</em>'
 
 
 def test_render_strong_marker_success():
     for text in sample_texts:
-        assert render_strong_marker(text) == f'<strong>{text}</strong>'
+        node = Span(**{'_type': 'span', 'text': text})
+        block = Block(_type='block', children=[node])
+        assert StrongMarkerDefinition.render(node, 'strong', block) == f'<strong>{text}</strong>'
 
 
 def test_render_link_marker_success():
     for text in sample_texts:
-        assert render_link_marker(text=text, href=text) == f'<a href="{text}">{text}</a>'
-
-    with pytest.raises(TypeError):
-        # enforce keyword-only arguments to limit possibility of error
-        render_link_marker('test', 'test')
+        node = Span(**{'_type': 'span', 'marks': ['linkId'], 'text': text})
```

Comment:
We probably don't need to use unpacking for these, thought changing them back might not be worth the trouble 🤷 
```suggestion
        node = Span(_type='span', marks=['linkId'], text=text)
```